docs/document/content/quick-start/shardingsphere-scaling-quick-start.cn.md

```diff
@@ -16,7 +16,7 @@ weight = 3
 
 如果后端连接 PostgreSQL 数据库，不需要引入额外依赖。
 
-如果后端连接 MySQL 数据库，请下载 [mysql-connector-java-5.1.47.jar](https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.47/mysql-connector-java-5.1.47.jar)，并将其放入 `%SHARDINGSPHERE_PROXY_HOME%/lib` 目录。
+如果后端连接 MySQL 数据库，请下载 [mysql-connector-java-5.1.47.jar] (https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.47/mysql-connector-java-5.1.47.jar)，并将其放入 `%SHARDINGSPHERE_PROXY_HOME%/lib` 目录。
```

Comment:
Please do not remove space between `[mysql-connector-java-5.1.47.jar]` and `(https://repo1.maven.org/maven2/mysql/mysql-connector-java/5.1.47/mysql-connector-java-5.1.47.jar)`, which is markdown syntax